astrosat_users/views/views_auth.py

```diff
@@ -114,7 +114,10 @@ def has_permission(self, request, view):
                 openapi.Schema(type=openapi.TYPE_STRING, example="superpassword23"),
             ),
             ("accepted_terms", openapi.Schema(type=openapi.TYPE_BOOLEAN)),
-            ("requires_customer_registration_completion", openapi.Schema(type=openapi.TYPE_BOOLEAN)),
+            (
+                "requires_customer_registration_completion",
+                openapi.Schema(type=openapi.TYPE_BOOLEAN),
+            ),
```

Comment:
This appears just to be a formatting change, likely caused by me introducing `Format on Save` on your IDE on Friday. Since we don't have `yapf`, I'm confused why this actually happened. It's no biggie, just concerned that from now on lot of files will include such changes, obscuring the real changes. Re-formatting, should be applied to all files in one commit, to avoid this.

Reply:
It is a formatting change.  I thought it was b/c of `yapf` -  I'll double-check this.